src/ontology/phenio-odk.yaml

```diff
@@ -11,129 +12,206 @@ export_formats:
   - owl
   - json
 allow_equivalents: all
+sssom_mappingset_group:
+  products:
+    - id: uberon
+      maintenance: mirror
+      mirror_from: http://purl.obolibrary.org/obo/uberon/uberon.sssom.tsv
+    - id: cl
+      maintenance: mirror
+      mirror_from: https://raw.githubusercontent.com/obophenotype/uberon/refs/heads/master/src/mappings/cl.sssom.tsv
```

Comment:
Err, why not getting the CL set from its “official” PURL? `http://purl.obolibrary.org/obo/cl/cl.sssom.tsv` ?

Reply:
Fixed, I thought I tried to find it but I didn't now I see. :)